source/catolog.html

```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Кэт энерджи</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link rel="stylesheet" href="styles/styles.css">
+  <script src="scripts/index.js" defer></script>
+</head>
+
+<body class="page__body">
+  <header class="main-header">
+    <a class="main-header__logo" href="#">
+      <img class="main-header__logo-image" src="#" alt="Изображение логотипа" width="24" height="11">
+      <img class="main-header__logo-image" src="#" alt="название логотипа" width="101" height="18">
+    </a>
+    <nav class=" main-header-main-menu  main-menu">
+      <button class="main-header__main-menu-toggle" type="button"><span class="visually-hidden">Открыть меню</span>
+      </button>
+      <div class="main-menu__container">
+        <ul class="main-menu__list">
+          <li class=" main-menu__item">
+            <a href="#">
+              Главная
+            </a>
+          </li>
+          <li class="main-menu__item main-menu__item--current">
+            <a href="#">
+              Католог продукции
+            </a>
+          </li>
+          <li class="main-menu__item">
+            <a href="#">
+              Подбор программы
+            </a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main class="main-container">
+    <section class="product-list">
+      <h1 class="catolog-inner-title">Каталог продукции</h1>
+      <div class="product-list__container">
+        <ul class="product-list__list">
+          <li class="product-list__item  product-card">
+            <img class="product-card__product-image" src="#" alt="Изображение товара" width="68" height="86">
+            <h3 class="product-card__title">Cat Energy PRO 500 г</h3>
+            <div class="product-card__product-features  product-features">
+              <span class="product-features__product-features-element">Масса</span>
+              <span class="product-features__product-features-value">500г</span>
```

Comment:
Давай сократим чуть (в следующих карточках аналогично).

```suggestion
              <span class="product-features__element">Масса</span>
              <span class="product-features__value">500г</span>
```